addons/account/models/partner.py

```diff
@@ -240,27 +240,26 @@ def _get_fiscal_position(self, partner, delivery=None):
         company = self.env.company
         eu_country_codes = set(self.env.ref('base.europe').country_ids.mapped('code'))
```

Comment:
Sorry, I forgot to ask earlier, but while we're at it, shouldn't we also use `_xmlid_to_res_model_res_id()` here to reduce the number of queries?

Reply:
Done, I also reverted the commit about the query count in tests